Quadriga/src/main/java/edu/asu/spring/quadriga/rest/NetworkRestController.java

```diff
@@ -663,5 +676,67 @@
         return new ResponseEntity<String>(res.toString(), httpHeaders, HttpStatus.OK);
 
     }
+    
+    
+    /**
+     * Rest interface to search concept in list of projects
+     * http://<<URL>:<PORT>>/quadriga/rest/sites/networks/search/{conceptId}/{projectIds}
+     * networks

```

Comment:
You javadoc should be more specific, e.g provide an example. Especially, how would the projectids be represented (comma-separated list, semicolon-separated list, ...)?

Also, you should give a description at least for the parameters "conceptId" and "projectIds" below.